### PR TITLE
feat: add preset to opt-out of rebase checkbox within PRs

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

Adds a preset that removes the `{{{control}}}` template from the PR.
This in essence removes the checkbox to "rebase this pr".
This can help when a user has CI that checks if all checkboxes within a PR are ticked.

## Context:

Inspired by #8387.

I'd feel better if we had a unit test to check that this preset actually works, and keeps working in the future.
We are migrating away from the rebase term, so a unit test would alert us if we're breaking the new preset.

Also please check if this preset is added to the right document.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — DISMISSED** [review]

File: lib/config/presets/internal/default.ts [hunk 1]
```diff
@@ -584,4 +584,10 @@ export const presets: Record<string, Preset> = {
     description: 'deprecated alias for config:js-lib',
     extends: ['config:js-lib'],
   },
+  disableRebaseCheckboxInPRs: {
```
Comment (rarkins) [inline lib/config/presets/internal/default.ts:587]:
```suggestion
  disableControls: {
```

File: lib/config/presets/internal/default.ts [hunk 2]
```diff
@@ -584,4 +584,10 @@ export const presets: Record<string, Preset> = {
...
+    description:
+      'Remove the checkbox to rebase the PR, this can help when your CI checks if all checkboxes are ticked.',
```
Comment (rarkins) [inline lib/config/presets/internal/default.ts:589]:
```suggestion
      'Remove the checkbox controls from PRs',
```

**Review by rarkins — CHANGES_REQUESTED** [review]

**Comment by HonkingGoose** [comment]
Seems the auto-merge functionality got stuck somehow, I've merged the latest `master` branch.
Let's see what happens now.

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
@HonkingGoose approval was missing. 🤗

**Comment by HonkingGoose** [comment]
@viceice Ah okay. 😄 I thought it was because my fork branch was not up-to-date with `renovate/master`... 🙃

**Comment by viceice** [comment]
This is another cause. All required checks need to be green. 🤗

**Pull request merged**